training/04_editing_eml.Rmd

```diff
@@ -12,6 +12,16 @@ This chapter will be longest of all the sections! This is a reminder to take fre
 ```{r, child = '../workflows/edit_eml/edit_an_eml_element.Rmd'}
 ```
 
+## FAIR data practices
+
+The result won't show up on the webpage but it should add a publisher element to the dataset element and a system to all of the entities based on what the PID is. This will help make our metadata more FAIR (Findable, Accessible, Interoperable, Reusable). 
```

Comment:
```suggestion
The result of these function calls won't show up on the webpage but they will add a `publisher` element to the `dataset` element and a `system` to all of the entities based on what the PID is. This will help make our metadata more FAIR (Findable, Accessible, Interoperable, Reusable). 
```